elytron/WFCORE-5120-client-default-ssl-context.adoc

```diff
@@ -0,0 +1,86 @@
+= WFCORE-5120 Automatic registration of client side / JVM wide default SSLContext
+:author:            Diana Vilkolak Krepinska
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This issue is about providing ability to register a client side JVM wide default SSLContext for the use of any libraries that support the use of the default SSL context.
```

Comment:
I think I don't get the exact meaning of this sentence.

What do you mean by libraries here? 
Could you make an example that could help the reader to "visualize" or at least "map" the proposed feature with respect to the current server behavior, e.g.: a use case?

What is the goal of the feature from the user perspective? 
Or there is no change from the user point of view (i.e. not a user facing feature)? 
If this is just an internal feature, how is the server behavior changing?

This is the overview section and its goal should be to give an high level picture of the proposed changes to the readers, independently from their knowledge or expertise, without for them to need to dig into the details.

For instance, try to make the context clear (i.e. this feature is security related, and affects the Elytron framework, specifically with respect to the SSL default context usage), its scope (it affects client/server code) and to describe the difference between the current server behavior and the one of the variant with the implemented feature (what is missing now compared to what the feature will bring).

Or just use the text that you've prepared for the feature community documentation, i.e. https://github.com/wildfly/wildfly/pull/14335/files#diff-e3a0588bb02dfdf26045958b2b759500698c626ae9b56209fa6233904120e0e9R295

Again not great details, which can be covered in the following sections, but enough information to understand the feature context.